src/avram/query_builder.cr

```diff
@@ -303,6 +303,17 @@ class Avram::QueryBuilder
     block.call(self)
   end
 
+  # Clears the last conjunction
+  # e.g. users.age = $1 AND -> users.age = $1
+  def clear_conjunction
+    @wheres.last.conjunction = Avram::Where::Conjunction::None unless @wheres.empty?
+  end
+
+  # Removes the last `Avram::Where` to be added
+  def remove_last_where
+    @wheres.pop
```

Comment:
Should this return `self`? 🤔

Reply:
@matthewmcgarvey what are your thoughts on this? I assume no one will call this but us... but should this return `self`? or do you think it even matters?

Reply:
I was looking at query builder yesterday and couldn't see any pattern to when we clone queryable vs query builder. I wouldn't worry about it

Reply:
Cool. Thanks for the reviews!